test/unit/ItemFilters.test.js

```diff
@@ -145,7 +145,7 @@ describe('ItemFilters', () => {
       expect(component.find('.item-filter-info').find('span').text()).to.equal("Filtered by format: 'Text'");
     });
     it('should display "5 Results Found"', () => {
-      expect(component.find('h3').text()).to.equal('5 Results Found');
+      expect(component.find('h3').text()).to.equal('5 Matching Items');
```

Comment:
can you update the string here to reflect the test expectation?

Reply:
Yes, good point that I missed.